### DISCUSSION TITLE
Questions on Using DIANN for TMTpro Searches

### DISCUSSION THREAD

**Discussion by sh18545** [vdemichev/DiaNN#1118] [Q&A]
Hello,

I'm the graduate student who mentioned having questions on Twitter. I am trying to perform a TMTpro search using DIANN with the following command: --fixed-mod "TMTpro,304.207146, N-term;TMTpro,304.207146, K"
However, it seems like the search is not being conducted properly. The following log message is displayed: Cannot find a UniMod modification match for TMTPro: 148.106 minimal mass discrepancy; using the original modification name
I believe this is the issue causing the problem. Could you provide any advice on how to resolve this?

Thank you for your help!

**Comment by vdemichev** [comment]
Hi,

Please see the syntax for --fixed-mod, e.g. the declaration for the N-term is via 'n'.
Note that reporter tag dissociation is not accounted for by DIA-NN in any way at the moment, i.e. multiplexing in DIA-NN is in general meant to be used for non-isobaric tags that don't release reporter ions.

> Cannot find a UniMod modification match for TMTPro

This is not an error.

Best,
Vadim

**Comment by sh18545** [comment]
Hi Vadim,

Thank you for your response. I understand now that the syntax for --fixed-mod needs to be precise. I will ensure to use 'n' for the N-terminal declaration. Also, I appreciate the clarification regarding the reporter tag dissociation and that this message is not an error.
Thank you for your assistance.

Best regards,
sh18545

**Comment by vyaspujari** [comment]
Hi Vadim,

Does DIA-NN support TMTproZero tags? I used the following in the additional options pane: --fixed-mod TMT,295.189592,nK

Still getting this warning:

Cannot find a UniMod modification match for TMT: 139.089 minimal mass discrepancy; using the original modification name

I am only quantifying ~70k peptides on a HeLa standard that is TMTproZero labeled, whereas in an unlabeled sample, I get >150k peptides

**Reply by vdemichev** [reply]
Hi,

Yes, it should work fine. Please use fine tuning https://github.com/vdemichev/DiaNN?tab=readme-ov-file#fine-tuning-prediction-models for optimal performance.

Best,
Vadim
